packages/appstore/src/components/containers/trading-app-card.tsx

```diff
@@ -141,7 +142,12 @@ const TradingAppCard = ({
             >
                 <TradingPlatformIconProps icon={icon} onClick={clickable_icon ? openStaticPage : undefined} size={48} />
             </div>
-            <div className={classNames('trading-app-card__container', { 'trading-app-card--divider': has_divider })}>
+            <div
+                className={classNames('trading-app-card__container', { 'trading-app-card--divider': has_divider })}
+                data-testid={`trading-app-card-${platform_name.replaceAll(' ', '-').toLowerCase()}${
```

Comment:
```suggestion
                data-testid={`dt_trading-app-card_${platform_name.replaceAll(' ', '-').toLowerCase()}${
```

Reply:
@mahdiyeh-deriv thanks Mahdiyeh for your suggestion, If I add demo and real here then the deriv platforms like binary bot or deriv trader will also get that label ? Also I see you suggesting Underscore for some parts should I follow any convention moving forward what is the pattern?

Reply:
@mitra-deriv it should be fine to get it in the platform cards as well since we have different trading-app-card for demo and real for this.
regarding the convention you can check the last line here: `packages/reports/README.md`

Reply:
sure noted and thanks 🙏